cfme/tests/v2v/test_v2v_migrations.py

```diff
@@ -225,3 +248,23 @@ def test_dual_datastore_dual_vm_mapping_crud(appliance, enable_disable_migration
     mapping = infrastructure_mapping_collection.create(form_data_dual_datastore)
     view = navigate_to(infrastructure_mapping_collection, 'All', wait_for_view=True)
     assert mapping.name in view.infra_mapping_list.read()
+
+
+@pytest.mark.parametrize('vm_list', ['NFS_Datastore_1', 'iSCSI_Datastore_1'], ids=['NFS', 'ISCSI'],
+                         indirect=True)
+@pytest.mark.parametrize('form_data_single_datastore', [['nfs', 'nfs']], indirect=True)
```

Comment:
**REQUIRED:** This parametrization does nothing right now as you only have a single parameter. Is this just a future proof effort so that we can spin up other similar tests in the future?

Reply:
@psav I believe that is exactly what @Yadnyawalkya meant to do. We will be adding more parameters in there for 'form_data_single_datastore' such as ['iscsi','iscsi'], ['iscsi','nfs'] and ['nfs','iscsi']